geode-core/src/distributedTest/java/org/apache/geode/cache30/GlobalRegionDUnitTest.java

```diff
@@ -71,71 +71,58 @@
-            .isTrue();
+    vm0.invoke("Create GLOBAL Region", () -> {
+      try {
+        createRegion(name, "INCOMPATIBLE_ROOT", getRegionAttributes());
+      } catch (CacheException ex) {
+        fail("While creating GLOBAL region", ex);
       }
+      assertThat(getRootRegion("INCOMPATIBLE_ROOT").getAttributes().getScope().isGlobal())
+          .isTrue();
     });
 
-    vm1.invoke(new SerializableRunnable("Create NO ACK Region") {
-      @Override
-      public void run() {
+    vm1.invoke("Create NO ACK Region", () -> {
+      try {
+        RegionFactory<Object, Object> factory =
+            getCache().createRegionFactory(getRegionAttributes());
+        factory.setScope(Scope.DISTRIBUTED_NO_ACK);
         try {
-          RegionFactory<Object, Object> factory =
-              getCache().createRegionFactory(getRegionAttributes());
-          factory.setScope(Scope.DISTRIBUTED_NO_ACK);
-          try {
-            assertThat(getRootRegion("INCOMPATIBLE_ROOT")).isNull();
-            createRegion(name, "INCOMPATIBLE_ROOT", factory);
-
-            fail("Should have thrown an IllegalStateException");
-          } catch (IllegalStateException ignored) {
-            // pass...
-          }
+          assertThat(getRootRegion("INCOMPATIBLE_ROOT")).isNull();
+          createRegion(name, "INCOMPATIBLE_ROOT", factory);
 
-        } catch (CacheException ex) {
-          fail("While creating GLOBAL Region", ex);
+          fail("Should have thrown an IllegalStateException");
+        } catch (IllegalStateException ignored) {
+          // pass...
         }
+
+      } catch (CacheException ex) {
+        fail("While creating GLOBAL Region", ex);
       }
     });
 
-    vm1.invoke(new SerializableRunnable("Create ACK Region") {
-      @Override
-      public void run() {
+    vm1.invoke("Create ACK Region", () -> {
+      try {
+        RegionFactory<Object, Object> factory =
+            getCache().createRegionFactory(getRegionAttributes());
+        factory.setScope(Scope.DISTRIBUTED_ACK);
         try {
-          RegionFactory<Object, Object> factory =
-              getCache().createRegionFactory(getRegionAttributes());
-          factory.setScope(Scope.DISTRIBUTED_ACK);
-          try {
-            Region<Object, Object> rootRegion = factory.create("INCOMPATIBLE_ROOT");
-            fail("Should have thrown an IllegalStateException");
-            factory.createSubregion(rootRegion, name);
-            fail("Should have thrown an IllegalStateException");
-
-          } catch (IllegalStateException ex) {
-            // pass...
-            assertThat(getRootRegion()).isNull();
-          }
-
-        } catch (CacheException ex) {
-          fail("While creating GLOBAL Region", ex);
+          Region<Object, Object> rootRegion = factory.create("INCOMPATIBLE_ROOT");
+          fail("Should have thrown an IllegalStateException");
+          factory.createSubregion(rootRegion, name);
```

Comment:
I don't think the code can ever reach lines 116 or 117

Reply:
true. Deleted.